crates/matrix-sdk-ui/src/timeline/event_item/content/pinned_events.rs

```diff
@@ -0,0 +1,178 @@
+use std::collections::HashSet;
```

Comment:
Missing copyright.

Reply:
Oh, weird. I expected the IDE to automatically add it 🤔 .